en/docs/integrations/integration-concepts.md

```diff
@@ -0,0 +1,98 @@
+   |                             | milestoned             | Add a milestone to an issue              |
+   |                             | demilestoned           | Remove a milestone from an issue         |
+   |                             | closed                 | Close an issue                           |
+   |                             | reopened               | Reopen an issue                          |
+   | label                       | created                | Create a label                           |
+   |                             | edited                 | Edit a label                             |
+   |                             | deleted                | Delete a label                           |
+   | milestone                   | created                | Create a milestone                       |
+   |                             | closed                 | Close a milestone                        |
+   |                             | opened                 | Open a milestone                         |
+   |                             | edited                 | Edit a milestone                         |
+   |                             | deleted                | Delete a milestone                       |
+   | pull_request                | assigned               | Assign a pull request to a user          |
+   |                             | unassigned             | Unassign a pull request from a user      |
+   |                             | review_requested       | Request a user to review a pull request  |
+   |                             | review_request_removed |                                          |
+   |                             | labeled                | Add a label to a pull request            |
+   |                             | unlabeled              | Remove a label from a pull request       |
+   |                             | opened                 | Open a pull request                      |
+   |edited                       | Edit a pull request    |                                          |
+   |                             | closed                 | Close a pull request                     |
+   |                             | reopened               | Reopen a pull request                    |
+   | pull_request_review         | submitted              | Submit a pull request                    |
+   |                             | edited                 | Edit a pull request review               |
+   |                             | dismissed              | Dismiss a pull request review            |
+   | pull_request_review_comment | created                | Create a pull request review comment     |
+   |                             | edited                 | Edit a pull request review comment       |
+   |                             | deleted                | Delete a pull request review comment     | 
+
+
+## Connection
```

Comment:
Similar to services concpets page, we need to have a section named `Statement` here as well right?

Reply:
Added content related to the `Statement` concept.